presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMultimapAggAggregation.java

```diff
@@ -130,4 +205,36 @@ public void testDoubleRowMap()
 
         assertAggregation(aggFunc, map.isEmpty() ? null : map, builder.build());
     }
+
+    private static <K, V> void testMultimapAggWithGroupBy(
+            InternalAggregationFunction aggregationFunction,
+            GroupedAccumulator groupedAccumulator,
+            int groupId,
+            Type keyType,
+            List<K> expectedKeys,
+            Type valueType,
+            List<V> expectedValues)
+    {
+        // Flatten out the expected output into two blocks which represent the keys and values to be aggregated
+        RowPageBuilder pageBuilder = RowPageBuilder.rowPageBuilder(keyType, valueType);
+        ImmutableMultimap.Builder<K, V> outputBuilder = ImmutableMultimap.builder();
+        for (int i = 0; i < expectedValues.size(); i++) {
+            pageBuilder.row(expectedKeys.get(i), expectedValues.get(i));
+            outputBuilder.put(expectedKeys.get(i), expectedValues.get(i));
+        }
+        Page page = pageBuilder.build();
+
+        AggregationTestInput input = new AggregationTestInputBuilder(
+                new Block[]{page.getBlock(0), page.getBlock(1)},
```

Comment:
style nit:

```java
        AggregationTestInput input = new AggregationTestInputBuilder(
                new Block[] {page.getBlock(0), page.getBlock(1)},
                aggregationFunction).build();
```

Reply:
I think we could add a Checkstyle rule to enforce that right parens don't have any preceeding whitespace.  What do you think?